### ISSUE TITLE
problem with afterColumnMove

### ISSUE THREAD

**Issue by beblaawy** [handsontable/handsontable#5173]
I have this order of columns:
![original-columns](https://user-images.githubusercontent.com/19722460/41245530-87734d58-6da8-11e8-9d6e-c366722f0627.png)

when I use `afterColumnMove: function(columns, target){}`
to move **sku** after **id** I got the target = 1 and this is a correct behavior

when I move the **sku** back to it's original position I got target = 5 and it should equal to 4

* Isn't the **target** the index of moved position ?
* Is there is a property to get the new arranged columns ?

**Comment by AMBudnik** [comment]
Hi @ZarBoXa 

the issue sounds like the one reported at https://github.com/handsontable/handsontable/issues/4470
Can you confirm?

**Comment by ZarBoXa** [comment]
@AMBudnik No it's not the same .. I see when move title to left It works just fine but when move to right for example after "Model Number" the target is 4 I think It count the indexes before moving ( 0 => id, 1 => Year, 2 => Title, 3 => Model Number, 4 => the new title )
is there any property that holds this arrangement ?

**Comment by AMBudnik** [comment]
Hi @ZarBoXa 

Yes, that is exactly how the indexes are currently added, but it can be confusing. We have created an epic at https://github.com/handsontable/handsontable/issues/5112 that, when applied, will change the hook and method logic for all operations that are working on visual and physical indexes. 
I will mark your case as a `bug` to make sure that it is also tested while adding the change in #5112

**Comment by aninde** [comment]
@beblaawy this issue will be fixed by the new `index mappers` in `8.0.0-rc1` v.
Changes will be released soon.

**Steps to reproduce:**
1. Move column E after column A.
2. Move column E to its original position, after column D.
3. Check console.

**Wrong result:**
After the first move: Target is 1 //correct
After the second move: target is 5 //incorrect

**Expected result - on `8.0.0-rc1`**
After first move: Target is 1 //correct
After second move: target is 4 //correct

**Minimalised demo**
v. `7.1.1` with bug https://jsfiddle.net/aninde/pfbe1vx5/
v. `8.0.0-rc1` fixed https://jsfiddle.net/aninde/nyv2t0hr/

**Comment by beblaawy** [comment]
Thank you guys!

**Comment by AMBudnik** [comment]
Please do not close the issue. We will close them all after final tests and the official release.

**Comment by AMBudnik** [comment]
Hi @beblaawy 

now you can test the beta, it is on the NPM `npm i handsontable@beta`

and the resources are

```js
</style><script src="https://cdn.jsdelivr.net/gh/handsontable/handsontable@release/8.0.0-beta.1/dist/handsontable.full.js"></script><link type="text/css" rel="stylesheet" href="https://cdn.jsdelivr.net/gh/handsontable/handsontable@release/8.0.0-beta.1/dist/handsontable.full.css">
```

Fixed demo https://jsfiddle.net/fsw216pq/

**Comment by AMBudnik** [comment]
Hi @beblaawy 

we've just released Handsontable v 8.0.0 that should solve this issue. Could you update to v 8.0.0 and confirm? I'd really appreciate it.

CDN https://cdn.jsdelivr.net/gh/handsontable/handsontable@latest/dist/handsontable.full.css
https://cdn.jsdelivr.net/gh/handsontable/handsontable@latest/dist/handsontable.full.js
NPM npm i handsontable
Release Notes https://github.com/handsontable/handsontable/releases

**Comment by AMBudnik** [comment]
Feel free to make an update when you'll be ready @beblaawy 
I'm closing this issue as officially solved.
